bot/cogs/utilities.py

```diff
@@ -2,22 +2,43 @@
 from pathlib import Path
 import random
 import subprocess
+import datetime
```

Comment:
```suggestion
from datetime import datetime
```
You only use `datetime.datetime` for `utcnow()`, I don't think adding an extra layer is necessary.